src/Java2/Main.java

```diff
@@ -3,6 +3,71 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+        //Массив участников забега
+        JumpRunnable[] arrJumpRunnable = {
+                new Human("Валера", 150, 1.2),
+                new Cat("Мурзик", 100, 1.5),
+                new Robot("R2D2", 1000, 3.0)
+        };
+
+        //массив препядствий
+        Obstacle[] arrObstacles = {
+                new Treadmill(100),
+                new Wall(1.25),
+                new Wall(1.1),
+                new Treadmill(200),
+                new Wall(1.4),
+                new Treadmill(990)
+        };
+
+        for (JumpRunnable j : arrJumpRunnable ) {
+
+            System.out.printf("\nНа полосу препядствий выходит %s\n", j.getName());
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.